### ISSUE TITLE
Add button to skip flight

### ISSUE THREAD

**Issue by jaredkhan** [OneZoom/OZtree#920]
Would suggest we add a button during flights (especially long flights) that lets you skip the flight.

At the moment if you want to go to a new location but not interested in the relationship to your current location, you have to go to Settings > Search Mode > Jump to results, then do the search, then turn the setting back to 'Fly to results' when you need it.

**Comment by hyanwong** [comment]
I completely agree with this. I thought maybe we could have two separate links from the search dropdown, one that jumps and one that flies, but I suspect your idea of a popup button would be more intuitive.

Do you have any idea how easy this would be to code up @jaredkhan ? This would be a nice project if you had any time to figure out how to do it.
